WMIAdventure/frontend/src/js/components/battle/organisms/BattleView/BattleView.js

```diff
@@ -453,6 +459,8 @@ class BattleView extends React.Component {
     // scale USER or ENEMY effect icons to signal effect action
     effectsActions = (userTurn, index = 0) => {
         const effects = this.state.battle.currentTurn.usedEffects;
+        console.log(effects);
+        this.visualizeEffects(effects);

```

Comment:
Zostawiłem nawet komentarz gdzie powinno się to wykonywać. Tutaj wywołują się wszystke efekty i to nie sprawdza w ogóle jaki efekt się wykonuje w danej chwili. W `effectIconCastEffect` jest komentarz i tam powinno się wykonywać wizualizowanie efektów